browser/main/lib/dataApi/createNoteFromUrl.js

```diff
@@ -14,12 +14,18 @@ function validateUrl (str) {
   }
 }
 
+const ERROR_MESSAGES = {
+  ENOTFOUND: 'URL not found. Please check the URL, your internet connection and try again.',
```

Comment:
I would rephrase this as:
`URL not found. Please check the URL, or your internet connection and try again.`

Reply:
That's better. I've updated it.